### DISCUSSION TITLE
why the popup window shown different language?

### DISCUSSION THREAD

**Discussion by summerfind** [microsoft/vscode-discussions#904] [Extension Show and Tell]
Hi all.
the vscode warning window shown as picture, how to fix it shown English?
Thanks
Adam

![photo_2023-11-01_10-42-33](https://github.com/microsoft/vscode-discussions/assets/84052634/c4289999-08db-4f92-83ea-349314560578)

**Comment by DhrubajyotiBiswasfounderofDribll** [comment]
Hi, @summerfind You change your VS Code language by following the steps from the official VS Code documentation:

- Open VS Code
- Press F1 and type 'display'
- Choose 'Configure display language'
- Select your language. On top are installed languages, and below are other languages. The one you select will be 
   installed as an extension.
- Accept to restart.
you can see that the language you choose has been added to the extensions by selecting 'extension' icon on the left toolbar (look for '#### language Pack')

I hope it really helped you!
